webapp/src/templates/partials/reports_content.html

```diff
@@ -6,50 +6,50 @@
     </div>
   </div>
 
-  <div class="col-sm-8 item-content empty-selection" ng-show="!loadingContent && !selected.length">
+  <div class="col-sm-8 item-content empty-selection" ng-show="!loadingContent && !$ctrl.selected.length">
     <div translate>No report selected</div>
   </div>
 
-  <div id="reports-content" class="col-sm-8 item-content" ng-show="$ctrl.selectMode || (!loadingContent && selected.length && selected[0].doc.form !== undefined)">
+  <div id="reports-content" class="col-sm-8 item-content" ng-show="$ctrl.selectMode || (!loadingContent && $ctrl.selected.length && $ctrl.selected[0].doc.form !== undefined)">
 
-    <div class="selection-count" ng-show="$ctrl.selectMode && selected.length">
-      <span ng-show="selected.length === 1" translate>select.mode.count.singular</span>
-      <span ng-show="selected.length !== 1" translate translate-values="{ number: {{selected.length}} }">select.mode.count.plural</span>
+    <div class="selection-count" ng-show="$ctrl.selectMode && $ctrl.selected.length">
+      <span ng-show="$ctrl.selected.length === 1" translate>select.mode.count.singular</span>
+      <span ng-show="$ctrl.selected.length !== 1" translate translate-values="{ number: {{$ctrl.selected.length || 0}} }">select.mode.count.plural</span>
     </div>
 
-    <div ng-repeat="selection in selected track by selection._id" ng-init="summary = selection.formatted || selection.summary; valid = selection.summary.valid || (selection.formatted && !selection.formatted.errors.length)" class="body report-body">
```

Comment:
This `ng-init` has been replaced by `$ctrl.summaries` and `$ctrl.validChecks` below. The `ng-init` approach doesn't work with redux as the store is immutable, and `ng-init` is only run on init, so `summary` and `valid` don't get updated when `$ctrl.selected` changes. I tried a lot of different approaches to make this work but kept running into the same problem of these only being assigned on init and then never updating. The solution that worked in the end was just making a selector for each of these properties and using `$index`, which is not very clean but I can't see another solution at the moment.